landoui/templates/stack/stack.html

```diff
@@ -104,7 +104,7 @@ <h2>Landing is blocked:</h2>
       <div class="StackPage-actions-headline">Preview Landing</div>
       <div class="StackPage-actions-subtitle">You must log in first</div>
     </button>
-  {% elif not series or dryrun is none %}
+  {% elif not user_has_phabricator_token() and (not series or dryrun is none) %}
```

Comment:
I think we may be overloading this button a little too much.

A few options:

- We could rename the button to "Landing Options" (instead of "Preview Landing")
- The we could separate the landing section from the uplift section in a way that one or the other could be hidden depending on the state of the stack
- At this point, we could keep the landing options button enabled if either of the two things are possible


Alternatively, and perhaps more cleanly, we could move the "request uplift" functionality outside of the landing modal altogether. If it's just a dropdown and button, what is the value of showing the landing preview anyway?

This check (as to whether something is landable or not) is going to get a lot more rigorous with the revision worker changes, and I'm hesitant to overload the functionality of this button.

Reply:
After discussion with glob, we would like to try and land this change to unblock the web UI pilot. I'm going to work on moving this button outside the landing modal and onto the base stack page, tracked with https://bugzilla.mozilla.org/show_bug.cgi?id=1801959.

Reply:
Actually, hold off on that plan for a moment.